#158792524 Design Mockups

PromasterGuru opened the pull request:
#### What does this PR do?
- Implements mockups for converge level up project

#### Description of Task to be completed?
- Implement Converge codelab mockup

#### How should this be manually tested?
- Clone the project: `git clone https://github.com/PromasterGuru/ConvergeCodelab.git`
- Navigate or cd to the project directory: `cd ConvergeCodelab`
- Navigate or cd to the wireframes directory: `cd wireframes`

#### Any background context you want to provide?
- Wireframe guides developers by simulating the flow of user actions.

#### What are the relevant pivotal tracker stories?
- [#159846703](https://www.pivotaltracker.com/story/show/159846703)
- [#158792524](https://www.pivotaltracker.com/story/show/158792524)
- [#159051218](https://www.pivotaltracker.com/story/show/159051218)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/39240075/53099281-878a3a80-3536-11e9-8b5d-2fba055f517c.png)

![image](https://user-images.githubusercontent.com/39240075/53100299-8d811b00-3538-11e9-9650-e6c3681d3b7a.png)

joeeasy commented:
Nice work, on the mockup the design, It can also be better quality too. Why not try using something like figma and use the material design components to build your mockup.
BTW you don't need the login screen.

PromasterGuru commented:
> Nice work, on the mockup the design, It can also be better quality too. Why not try using something like figma and use the material design components to build your mockup.
> BTW you don't need the login screen.

I realized the software I was using didn't have material design features. I will try using some other mockup software to redesign the mockups before the end of today.

PromasterGuru committed:
feat(design-mockups): Design Mockups

- Implement mockups for the converge codelab app
- Update README.md with wireframes information.

[Delivers #158792524]

joeeasy commented:
LGTM, @PromasterGuru  I am merging this now
